docs/reference/ingest/processors/csv.asciidoc

```diff
@@ -13,6 +13,7 @@ Extracts fields from CSV line out of a single text field within a document. Any
 | `quote`           | no        | "        | Quote used in CSV, has to be single character string
 | `ignore_missing`  | no        | `true`   | If `true` and `field` does not exist, the processor quietly exits without modifying the document
 | `trim`            | no        | `false`  | Trim whitespaces in unquoted fields
+| `empty_value`     | no        | -        | Value used to fill empty fields, empty fields will be skipped if this is not provided
```

Comment:
Could we add a quick mention here what we consider empty fields? ("no value" and "empty quotes"). This way we have it spelled out which fields are officially considered empty.

Reply:
Good idea, I've added it